esp/src/src-react/components/InfoGrid.tsx

```diff
@@ -85,11 +86,11 @@ export const InfoGrid: React.FunctionComponent<InfoGridProps> = ({
                 formatter: React.useCallback(function (Message, idx) {
                     const info = extractGraphInfo(Message);
                     if (info.graphID && info.subgraphID && info.activityID) {
-                        const txt = "Graph " + info.graphID + "[" + info.subgraphID + "], " + info.activityName + "[" + info.activityID + "]";
-                        return <Link href={`#/workunits/${wuid}/metrics/sg${info.subgraphID}`}>{txt}</Link>;
+                        const txt = `Graph ${info.graphID}[${info.subgraphID}], ${info.activityName} [${info.activityID}]`;
+                        return <><Link href={`#/workunits/${wuid}/metrics/sg${info.subgraphID}`}>{txt}</Link>{info.message && <span>{info.message}</span>}</>;
```

Comment:
@GordonSmith Excluding the message in these two graph related cases seemed deliberate, so I wasn't sure if this ticket might just be closed as a won't fix or something? But it was a quick change, so I thought I'd go ahead and open the PR